carriage-user-ios/carriage-user-ios/SettingsLauncher.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  SettingsLauncher.swift
+//  carriage-user-ios
+//
+//  Created by Kinjal Jasani on 11/25/19.
+//  Copyright © 2019 Beth Mieczkowski. All rights reserved.
+//
+
+import UIKit
```

Comment:
Is there any reason why we're importing UIKit and not SnapKit?

Reply:
import UIKit is the default statement when you create a new file. I didn't need SnapKit in this particular file so I didn't import it.